docs/make.jl

```diff
@@ -34,6 +34,7 @@ pages = Any[
         "AtmosModel" => "Atmos/Model/AtmosModel.md",
         "Turbulence" => "Atmos/Model/turbulence.md",
         "Tracers" => "Atmos/Model/tracers.md",
+        "EX1: Rising Bubble" => "Atmos/Model/risingbubble.md",
```

Comment:
```suggestion
        "Examples" => Any[
            "Rising Bubble" => "Atmos/Model/risingbubble.md",
        ],
```

Reply:
I think the @charleskawczynski 's `skeleton` #986 provides some dir structure to this ; this is just a placeholder right now